src/lib/isStrongPassword.js

```diff
@@ -4,7 +4,7 @@ import assertString from './util/assertString';
 const upperCaseRegex = /^[A-Z]$/;
 const lowerCaseRegex = /^[a-z]$/;
 const numberRegex = /^[0-9]$/;
-const symbolRegex = /^[-#!$@%^&*()_+|~=`{}\[\]:";'<>?,.\/ ]$/;
+const defaultSymbols = '-#!$@%^&*()_+|~=`{}\\[\\]:";\'<>?,.\\/ ';
```

Comment:
A little question about this regex, i know that you just used the existing `symbolRegex` but is there any special reason for not matching the Back slash character `\` ?

Reply:
Really Idk.
@profnandaa what do you think?